packages/babel-plugin-ember-test-metadata/utils.js

```diff
@@ -56,48 +56,73 @@ function getProjectConfiguration(project) {
   return parsedProjectConfiguration;
 }
 
-function _getParsedClassicFilepath(pathSegments, projectConfiguration) {
-  const projectNamePathSeparators = projectConfiguration.pkg.name.split(path.sep);
+function _getClassicPath(preformattedPath, projectName) {
```

Comment:
Let's order these params logically from their scope of what they represent, wider scope (project) to narrow scope (filename). I also think we can make the naming a bit easier to understand.

```suggestion
function _getRelativePathForClassic(projectName, filePath) {
```